synapse/lib/stormsvc.py

```diff
@@ -182,6 +182,19 @@ async def _onTeleLink(self, proxy):
         names = [c.rsplit('.', 1)[-1] for c in clss]
 
         if 'StormSvc' in names:
+            rver = proxy._getSynVers()
+            if rver < (2, 8, 0):
+                mesg = f'Storm service {self.name} ({self.iden}) is running an older ' \
+                       f'version of synapse {rver}, please update to ensure proper function!'
+                logger.exception(mesg)
+
+                self.proxy._t_conf['retrysleep'] = 2.0
```

Comment:
The badversion would be thrown by the _runSvcInit call, right? shouldn't we catch that and set the retry time down here?